src/crimson/osd/heartbeat.cc

```diff
@@ -33,7 +33,10 @@ Heartbeat::Heartbeat(const crimson::osd::ShardServices& service,
     front_msgr{front_msgr},
     back_msgr{back_msgr},
     // do this in background
-    timer{[this] { (void)send_heartbeats(); }}
+    timer{[this] {
```

Comment:
nit, might want to revise the title, like

> crimson/osd/heartbeat: report peer failures to monitor

Reply:
ok:-)